be/CMakeLists.txt

```diff
@@ -928,29 +937,40 @@ if (${WITH_TENANN} STREQUAL "ON")
 endif()
 
 # Add all external dependencies. They should come after the starrocks libs.
-set(STARROCKS_LINK_LIBS ${STARROCKS_LINK_LIBS} ${STARROCKS_DEPENDENCIES})
+# static link gcc's lib
+set(STARROCKS_STD_DEPENDENCIES
```

Comment:
this reverts the PR fix for UT ASAN issue: https://github.com/StarRocks/starrocks/pull/49756/files